FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_UDP_Mode_CLI_Windows_Simulator/WinPCap/arch.c

```diff
@@ -0,0 +1,377 @@
+/*
+    FreeRTOS V7.3.0 - Copyright (C) 2012 Real Time Engineers Ltd.
+
```

Comment:
This is for sure not the right header!

Reply:
I removed the WinPCap folder altogether.